contracts/tests/test_deposit_locker.py

```diff
@@ -220,11 +138,43 @@ def withdraw(self, depositor):
     def can_withdraw(self, depositor):
         return self.deposit_locker.functions.canWithdraw(depositor).call()
 
+    def balance_of(self, address):
+        return get_balance_function(self.deposit_locker.web3, self.token_contract)(
+            address
+        )
+
 
-@pytest.fixture(scope="session")
-def testenv(deploy_contract, accounts, web3, release_timestamp):
+@pytest.fixture(scope="session", params=["ETHDepositLocker", "TokenDepositLocker"])
+def testenv(
+    request,
+    deploy_contract,
+    accounts,
+    web3,
+    release_timestamp,
+    premint_token_address,
+    premint_token_value,
+):
     """return an initialized Env instance"""
-    deposit_locker = deploy_contract("DepositLocker")
+    token_contract = None
+    constructor_args = ()
+    if request.param == "TokenDepositLocker":
+        token_name = "Trustlines Network Token"
+        token_symbol = "TLN"
+        token_decimal = 18
+        constructor_args = (
+            token_name,
+            token_symbol,
+            token_decimal,
+            premint_token_address,
+            premint_token_value,
+        )
+
+        token_contract = deploy_contract(
+            "TrustlinesNetworkToken", constructor_args=constructor_args
+        )
+        constructor_args = (token_contract.address,)
```

Comment:
I would make the difference between deposit_constructor_args and token_constructor_args

Reply:
outdated now, as there is not deposit constructor anymore.